execution-without-ci.mdx

```diff
@@ -26,6 +24,14 @@ The command will return a response with a link to your test report and some othe
 
 If you follow this link, you can find the test report containing all test results of your execution.
 
+### Parameters
```

Comment:
all of these are btw ALREADY explained in the API-reference, shouldn't we just LINK to it instead of duplicating it? Imho the api reference should be the single source of truth